src/use_case/display_tagged_recipe/DisplayTaggedOutputBoundary.java

```diff
@@ -0,0 +1,12 @@
+package use_case.display_tagged_recipe;
+
+import entity.Recipe;
+import java.util.List;
+
+public interface DisplayTaggedOutputBoundary {
+    /**
+     * Prepares the success view for the user.
+     * @param recipes the recipes to display.
+     */
+    void prepareSuccessView(List<Recipe> recipes);
```

Comment:
This should take a `DisplayTaggedOutputData` object instead of a `List<Recipe>`.